client/ide/lib/views/statusbar/idestatusbar.coffee

```diff
@@ -148,11 +147,13 @@ module.exports = class IDEStatusBar extends kd.View
 
   startSession: ->
 
-    kd.singletons.appManager.tell 'IDE', 'startCollaborationSession', (err) =>
-      @resetProgress()  if err
```

Comment:
callback is removed since it's no longer supported in `startCollaborationSession` method